### PR TITLE
Add unfollow script

### PR DESCRIPTION
**Pull request by sergeygorky**

### CODE REVIEW

File: unfollowtw.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            friends_cache.append(line.strip())
+
+    list_friends = [x for x in friends if x not in friends_cache]
+
+    f = open('friends_cache_{}.txt'.format(currrent_user), 'a')
+
+    for fr in range(limit):
+        if len(api.followers_ids(list_friends[fr])) < 200:
+            api.destroy_friendship(list_friends[fr])
+            print("Unfollow " + list_friends[fr])
```
Comment (Wasper256) [inline unfollowtw.py]: please use logging, instead of print

File: get_friends_list.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import click
+import tweepy
+
+
+@click.command()
+@click.option('--consumer_key', default=os.environ.get('CONSUMER_KEY'),
+              help='CONSUMER_KEY')
+@click.option('--consumer_secret', default=os.environ.get('CONSUMER_SECRET'),
+              help='CONSUMER_KEY')
+@click.option('--access_token', default=os.environ.get('ACCESS_TOKEN'),
```
Comment (Wasper256) [inline get_friends_list.py]: please remove getting keys and tokens from cli(you can get it from PR 14)

File: unfollowtw.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+@click.command()
+@click.option('--consumer_key', default=os.environ.get('CONSUMER_KEY'),
+              help='CONSUMER_KEY')
+@click.option('--consumer_secret', default=os.environ.get('CONSUMER_SECRET'),
+              help='CONSUMER_KEY')
+@click.option('--access_token', default=os.environ.get('ACCESS_TOKEN'),
+              help='ACCESS_TOKEN')
+@click.option('--access_token_secret',
+              default=os.environ.get('ACCESS_TOKEN_SECRET'),
+              help='ACCESS_TOKEN_SECRET')
```
Comment (Wasper256) [inline unfollowtw.py]: same, please remove getting keys and tokens from cli

File: get_friends_list.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        retry_errors=5
+    )
+
+    currrent_user = api.me().screen_name
+    friends_id = api.friends_ids(currrent_user)
+
+    with open('friends_{}.txt'.format(currrent_user), 'w') as f:
+        for f_id in friends_id:
+            name = api.get_user(f_id).screen_name
+            print("Write " + name)
```
Comment (Wasper256) [inline get_friends_list.py]: please use logger

File: unfollowtw.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            friends.append(line.strip())
+
+    friends_cache = []
+    with open('friends_cache_{}.txt'.format(currrent_user), 'r') as f:
+        for line in f:
+            friends_cache.append(line.strip())
+
+    list_friends = [x for x in friends if x not in friends_cache]
+
+    f = open('friends_cache_{}.txt'.format(currrent_user), 'a')
```
Comment (Wasper256) [inline unfollowtw.py]: please use ```with open``` construction to avoid unclosed file in case of errror

File: unfollowtw.py [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+                try:
+                    if len(api.followers_ids(list_friends[fr])) < 200:
+                        api.destroy_friendship(list_friends[fr])
+                        logger.info("Unfollow %s", list_friends[fr])
+                        f.write(list_friends[fr] + ' \n')
+                    else:
+                        logger.info("Add %s to file", list_friends[fr])
+                        f.write(list_friends[fr] + ' \n')
+                except:
+                    pass
```
Comment (Wasper256) [inline unfollowtw.py]: please add logger exeption, instead of pass

File: unfollowtw.py [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+        for fr in range(limit):
+            try:
+                if api.get_user(list_friends[fr]).friends_count < 200:
+                    api.destroy_friendship(list_friends[fr])
+                    logger.info("Unfollow %s", list_friends[fr])
+                    f.write(list_friends[fr] + ' \n')
+                else:
+                    logger.info("Add %s to file", list_friends[fr])
+                    f.write(list_friends[fr] + ' \n')
+            except:
```
Comment (Wasper256) [inline unfollowtw.py]: add some logging data in case of except

File: unfollowtw.py [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+                    try:
+                        if len(api.followers_ids(fr)) < 200:
+                            api.destroy_friendship(fr)
+                            logger.info("Unfollow %s", fr)
+                            f.write(fr + ' \n')
+                        else:
+                            logger.info("Add %s to file", fr)
+                            f.write(fr + ' \n')
+                    except:
+                        logger.ERROR("Pass second try!")
```
Comment (Wasper256) [inline unfollowtw.py]: i think here will be better to use `logger.exception('')`, so you can get debug information after fail

Comment (Wasper256) [reply unfollowtw.py]: and in other cases of exception too

Comment (sergeygorky) [reply unfollowtw.py]: Done

**Pull request merged**